--- inst.sh	Mon Dec  3 04:32:24 2001
+++ inst.sh	Sun Dec  9 10:29:25 2001
@@ -47,7 +47,7 @@
 #
 # Make sure this script is being run by the superuser.
 #
-if [ "${LOGNAME}" != "root" ]; then
+if [ `id -u` != "0" ]; then
     echo
     echo "You MUST run this script as superuser!"
     echo
@@ -58,30 +58,9 @@
     exit 1
 fi
 
+. CONFIG
 
-if [ ! -f /etc/ltsp.conf ]; then
-    echo
-    echo "Couldn't find /etc/ltsp.conf, probably indicates that"
-    echo "lts_core hasn't been installed yet"
-    echo
-    exit 1
-fi
-
-. /etc/ltsp.conf
-
-if [ ! -d ${LTSP_DIR} ]; then
-    echo
-    echo "Could not find ${LTSP_DIR}"
-    echo "Has it moved, without updating /etc/ltsp.conf ?"
-    echo
-    exit 1
-fi
-
-if [ ! -d /tftpboot/lts ]; then
-    mkdir -p /tftpboot/lts
-fi
-
-cp vmlinuz-2.4.19-ltsp-1      /tftpboot/lts/vmlinuz-2.4.19-ltsp-1
+cp vmlinuz-2.4.19-ltsp-1      ${DESTDIR}/tftpboot/lts/vmlinuz-2.4.19-ltsp-1
 
 find i386 -print | cpio -pmud --quiet ${LTSP_DIR}
 RS=$?

