--- inst.sh	Mon Nov  4 06:36:38 2002
+++ inst.sh	Tue Feb  4 21:12:58 2003
@@ -33,7 +33,7 @@
 #
 # Make sure this script is being run by the superuser.
 #
-if [ "${LOGNAME}" != "root" ]; then
+if [ `id -u` != "0" ]; then
     echo
     echo "You MUST run this script as superuser!"
     echo
@@ -44,15 +44,7 @@
     exit 1
 fi
 
-if [ ! -f /etc/ltsp.conf ]; then
-    echo 
-    echo "Couldn't find /etc/ltsp.conf, probably indicates that"
-    echo "lts_core hasn't been installed yet"
-    echo
-    exit 1
-fi
-
-. /etc/ltsp.conf
+. ./CONFIG
 
 if [ ! -d ${LTSP_DIR} ]; then
     echo 
@@ -71,6 +63,6 @@
     exit 1
 fi
 
-/sbin/ldconfig -r ${LTSP_DIR}/i386
+#/sbin/ldconfig -r ${LTSP_DIR}/i386
 
 
